WPFHospitalEditor/AdditionalInformation.xaml.cs

```diff
@@ -97,27 +97,51 @@ private void SetNameCommonAttributes()
 
         private void BtnEquipment_Click(object sender, RoutedEventArgs e)
         {
-            EquipmentWindow equipment = new EquipmentWindow(EquipmentToContentRows(), "=", role);
+            EquipmentAndMedicationWindow equipment = new EquipmentAndMedicationWindow(EquipmentToContentRows());
             equipment.ShowDialog();
         }
+        private void BtnMedications_Click(object sender, RoutedEventArgs e)
+        {
+            EquipmentAndMedicationWindow medications = new EquipmentAndMedicationWindow(MedicationsToContentRows());
+            medications.ShowDialog();
+        }
 
         private string[] EquipmentToContentRows()
         {
             string[] contentRows = new string [allEquipment.Count()];
             for (int i = 0; i < allEquipment.Count(); i++)
             {
-                contentRows[i] = allEquipment.ElementAt(i).Name + "=" + allEquipment.ElementAt(i).Quantity;
+                contentRows[i] = allEquipment.ElementAt(i).Name + AllConstants.contentSeparator + allEquipment.ElementAt(i).Quantity;
             }
             return contentRows;
         }
 
-        private Boolean IsReadOnly()
+        private string[] MedicationsToContentRows()
         {
-            if (role == Role.Patient)
+            string[] contentRows = new string[allMedications.Count()];
+            for (int i = 0; i < allMedications.Count(); i++)
             {
-                return true;    
+                contentRows[i] = allMedications.ElementAt(i).Name + AllConstants.contentSeparator + allMedications.ElementAt(i).Quantity;
             }
+            return contentRows;
```

Comment:
MedicationsToContentRows() and EquipmentToContentRows() methods has same logic and probably can be merged into one method.

Reply:
I saw that, and I tried to merge these functions, but they need different parameters(MedicationsToContentRows() needs IEnumerable<MedicationDto>, while EquipmentToContentRows() needs IEnumerable<EquipmentDto>.

Reply:
Yeah I didn't saw that. We can solve this later if needed.